x-pack/plugins/ml/server/routes/schemas/results_service_schema.ts

```diff
@@ -26,6 +26,7 @@ export const anomaliesTableDataSchema = schema.object({
   maxRecords: schema.number(),
   maxExamples: schema.maybe(schema.number()),
   influencersFilterQuery: schema.maybe(schema.any()),
+  actualPlotFunction: schema.maybe(schema.nullable(schema.string())),
```

Comment:
As above, maybe this should be renamed to `functionDescription` to match the name of the property in the records?

Reply:
Updated here https://github.com/elastic/kibana/pull/81662/commits/cb2429e192079f4e9a1cfc0e401a2a9544d19f23